paymentsheet/src/main/java/com/stripe/android/paymentsheet/PaymentOptionResult.kt

```diff
@@ -27,8 +29,11 @@ internal sealed class PaymentOptionResult(
 
     @Parcelize
     data class Canceled(
-        val mostRecentError: Throwable?
-    ) : PaymentOptionResult(Activity.RESULT_CANCELED)
+        val mostRecentError: Throwable?,
```

Comment:
This is relevant on Succeeded too, they might have deleted before selecting one. And maybe also on Failed?

Reply:
Good call, I will add it there too